plugins/kubernetes-backend/src/cluster-locator/MultiTenantConfigClusterLocator.test.ts

```diff
@@ -16,6 +16,7 @@
 
 import '@backstage/backend-common';
 import { MultiTenantConfigClusterLocator } from './MultiTenantConfigClusterLocator';
+import { AuthTokens } from '..';
```

Comment:
Best make this more specific to avoid import cycles, as in avoid importing from a direct ancestor

Reply:
Could you clarify what you mean here/what's an example of a more specific import?